cloud-storage.go

```diff
@@ -38,7 +39,7 @@ func NewCloudStorage(
 	gcpStorageEmulatorHost string,
 ) (CloudStorage, error) {
 	switch bucketProvider {
-	case "aws":
+	case "", "aws":
```

Comment:
is this a hacky way to handle our CI/CD pipeline? 😂

Reply:
@bahrunnur correct :+1: